### ISSUE TITLE
Hello after publishing the asset i still get this error

### ISSUE THREAD

**Issue by odion-cloud** [munafio/chatify#63]
Target class [App\Http\Controllers\vendor\Chatify\MessagesController] does not exist.

**Comment by odion-cloud** [comment]
fixed from controller name space
namespace App\Http\Controllers\vendor\Chatify;

**Comment by munafio** [comment]
@odion-cloud Please do not OPEN a new issue if you already opened one before related to the same issue!!
You did open 3 issues related to the same one!!
